### PR TITLE
DDCYLS-5418/added feature flag

### PR DESCRIPTION
**Pull request by sauravkitukale**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -67,7 +71,7 @@ mongodb {
 
 urls {
   login         = "http://localhost:9949/auth-login-stub/gg-sign-in"
-  loginContinue = "http://localhost:9000/digital-service-tax-registration-frontend"
+  loginContinue = "http://localhost:8744/digital-services-tax-registration"
```
Comment (tapiwa-tiyemba) [inline conf/application.conf:74]: I think the url was changed back to "service" instead of "services"

File: conf/application.conf [hunk 2]
```diff
@@ -82,5 +86,6 @@ tracking-consent-frontend {
 }
 
 features {
-  welsh-translation: true
+  welsh-translation = true
+  dstNewRegistrationFrontendEnable = false
```
Comment (tapiwa-tiyemba) [inline conf/application.conf:90]: this needs to be true. I believe it should only be false for production config

File: conf/application.conf (hunk 1 above)
Comment (sauravkitukale) [reply conf/application.conf:74]: this change should not be showing up here and i think its due to my rebase. 
Main branch :   loginContinue = "http://localhost:8744/digital-services-tax-registration

File: conf/application.conf (hunk 2 above)
Comment (sauravkitukale) [reply conf/application.conf:90]: ill updated this while testing i kept it false, and missed to change it back.

**Review by ashwinirudrappa — DISMISSED** [review]

File: test/controllers/actions/AuthActionSpec.scala [hunk 3]
```diff
@@ -147,14 +175,100 @@ class AuthActionSpec extends SpecBase {
...
           status(result) mustBe SEE_OTHER
-          redirectLocation(result) mustBe Some(routes.UnauthorisedController.onPageLoad().url)
+          redirectLocation(result) mustBe Some(
+            routes.UnauthorisedController.onPageLoad().url
+          )
+        }
+      }
+    }
+
+    "Able to land on new regisration with feat flag true" - {
```
[the remark this answers is not in the document]
Comment (sauravkitukale) [reply test/controllers/actions/AuthActionSpec.scala]: Updated.

**Review by ashwinirudrappa — APPROVED** [review]

**Pull request merged**